Sources/PointFree/TODO.swift

```diff
@@ -16,19 +16,6 @@ import Tagged
 import Tuple
 import UrlFormEncoding
 
-extension Tagged where Tag == EncryptedTag, RawValue == String {
```

Comment:
moved to `Views`. seems weird to do that, it could be in its own module. we also need to get the `encrypted` and `decrypted` helpers out of `HttpPipeline`, doesn't make a lot of sense to be in there.